src/BenchmarksClient/Workers/SignalRWorker.cs

```diff
@@ -144,6 +156,16 @@ public SignalRWorker(ClientJob job)
                             }
                         }
                         break;
+                    case "echoIdle":
+                        while (!cts.IsCancellationRequested)
+                        {
+                            await Task.Delay(_sendDelay);
+                            for (var i = 0; i < _connections.Count; i++)
+                            {
+                                await _connections[i].SendAsync("Echo", DateTime.UtcNow);
```

Comment:
Echo is no longer a `SendAsync` function, it returns a DateTime look at https://github.com/aspnet/benchmarks/pull/454/files#diff-fa312acc610ac75125885193f1fbe849R143 for example